app/services/resource_recommendation.rb

```diff
@@ -0,0 +1,66 @@
+# frozen_string_literal: true
+class ResourceRecommendation
+  def initialize(moment)
+    @moment = moment
+    @moment_keywords = []
+    @matched_resources = []
+  end
+
+  def get_keywords(array)
+    array.each do |item|
+      @moment_keywords.push(item['name'].split,
+                            html_clean(item['description']).split)
+    end
+  end
+
+  def modify_keywords
+    @moment_keywords = @moment_keywords.flatten
+    @moment_keywords.each do |keyword|
+      keyword.gsub!(%r{([_@#!%()\-=;><,{}\~\[\]\./\?\"\*\^\$\+\-]+)}, '')
```

Comment:
```suggestion
      keyword.gsub!(/[^a-z]+/i, '')
```

Instead of passing several special characters, you can match any character that is not a letter

`^` menas `not`
`[a-z]` means any letter
`+` mens one or more
The `i` at the end means case-insensitive

This is a kind of a JSbin for Relugar expressions. You can test that regex there:
 https://rubular.com/r/8NkkIIHtxUBUxv

Reply:
I like your simpler way better, especially since it would cover any special characters not present in my current regex expression.  However, if we only allow a-z characters, would that not affect special alphabetical characters such as é or ñ? This project is available in several languages, including Spanish, which could pose a problem for matching words.

Reply:
That's a good point. Fortunately Ruby has regex support for Unicode characters, such as `[[:alpha:]]`

https://ruby-doc.org/core-2.7.0/Regexp.html#class-Regexp-label-Character+Classes